int8 motor_id # control both motors with 0, motor 1 with 1, and motor 2 with 2
int16 pwm